WireTesting/Sources/WireTesting/SnapshotHelper.swift

```diff
@@ -0,0 +1,295 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SnapshotTesting
+import SwiftUI
+import XCTest
+
+/// A helper object for verifying image snapshots.
+///
+/// Create variations of the snapshot behavior using the "with" methods.
+
+public struct SnapshotHelper {
```

Comment:
This isn't new code but moved code right?

Reply:
Was it deleted from its old location? I can't see that it has been deleted

Reply:
Good catch, this is a left-over from jumping branches. But since it will be put here in a follow-up pr, I prefer to keep it now.